client/client-multi/client-cpp/kaa/configuration/storage/IConfigurationStorage.hpp

```diff
@@ -42,18 +38,18 @@ class IConfigurationStorage {
      *
      * @param bytes Configuration binary data.
      */
-    virtual void saveConfiguration(byte_buffer &&bytes) = 0;
+    virtual void saveConfiguration(std::vector<std::uint8_t>&& bytes) = 0;
 
     /**
      * Specifies routine to load configuration data.
      *
      * @return Configuration binary data.
      */
-    virtual byte_buffer loadConfiguration() = 0;
+    virtual std::vector<std::uint8_t> loadConfiguration() = 0;
 };
 
-}  // namespace kaa
+typedef std::shared_ptr<IConfigurationStorage> IConfigurationStoragePtr;
```

Comment:
just proposal :may be it's time to use aliases to reduce redundant code in the places where it's used ?